frontend/src/my_team/MyTeamView.jsx

```diff
@@ -4,25 +4,35 @@ import { useState, useContext } from "react";
 import { UserContext } from "../UserContext.js";
 import PlayerCard from "./PlayerCard.jsx";
 import ScoutOpponent from "../opponent/ScoutOpponent.jsx";
-import { useQuery } from "@tanstack/react-query";
+import { useQuery, useQueryClient } from "@tanstack/react-query";
 import { AppLoader } from "../suspense/AppLoader.jsx";
 
 function MyTeamView() {
   const PORT = import.meta.env.VITE_BACKEND_PORT;
   const userContext = useContext(UserContext);
+  const queryClient = useQueryClient();
 
   const [displayScout, setDisplayScout] = useState(false);
-  const [recommendations, setRecommendations] = useState({
-    bestPlayers: [],
-    response: { keyPoints: [], areasOfImprovement: [] },
+
+  const recommendations = useQuery({
+    queryKey: ["recommendations"],
+    queryFn: async () => {
+      const queryUrl = new URL(`${PORT}/scoutOpponent`);
+      queryUrl.searchParams.append("userId", userContext.user.id);
+      const response = await fetch(queryUrl);
+      const results = await response.json();
+      return results;
+    },
   });
 
+  const [style, setStyle] = useState("outsideOffenseScore");
   const myTeamPlayers = useQuery({
```

Comment:
https://github.com/user-attachments/assets/58a8442b-5a65-4ec6-828f-0c77021c1b33

The style state is changed in the ScoutOpponent child component, which calls this object's refetch method.